### ISSUE TITLE
Convert an in memory table to an out of core table 

### ISSUE THREAD

**Issue by Gyslain** [JuliaComputing/JuliaDB.jl#230]
I'm merging JuliaDB tables (these tables come from a data flux) into a global ndsparse table. But I'm running out of memory so I plan to use a "chunked out-of-core" table.
Is there a way to convert my table to an out of core table?
The JuliaDB mention the loadtable function to create out of core table but it  only reads csv files from disk.

**Comment by Gyslain** [comment]
The loadtable function has 2 options : chunk and output. The output option is used to store the table on disk.
Whereas distribute has only the chunk option, so I guess the table remains in memory. Am I wrong?

**Comment by joshday** [comment]
You're right, but you can then `save` it.  My understanding is that the following are more-or-less equivalent

```julia
loadndsparse(path, chunks=4, output="outdir")
```
---
```julia
nd = loadndsparse(path)

dnd = distribute(nd, 4)

save(dnd, "outdir")
```

**Comment by Gyslain** [comment]
Ok I see. After saving the chunked table, it may reside on the disk or in memory. Later, I can push new data into this table without worrying about memory.
Thanks for your help, we can close this task.
